### PR TITLE
feat(route): 支持上海市文旅局审批公告

### PR DESCRIPTION
**Pull request by gideonsenku**
## 该 PR 相关 Issue / Involved Issue

Close #

## 路由地址示例 / Example for the Proposed Route(s)

```routes
/gov/shanghai/wgj
```

## 新 RSS 路由检查表 / New RSS Route Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [x] 全文获取 fulltext
  - [x] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/gov/radar.js [hunk 1]
```diff
@@ -948,6 +948,14 @@ module.exports = {
                 target: (params, url) => `/gov/shanghai/yjj/${new URL(url).match(/yjj\.sh\.gov\.cn\/(.*)\/index.html/)[1]}`,
             },
         ],
+        'wsbs.wgj': [
+            {
+                title: '上海市文旅局审批公告',
+                docs: 'https://docs.rsshub.app/government.html#shang-hai-shi-ren-min-zheng-fu-shang-hai-shi-wen-lv-ju-shen-pi-gong-gao',
+                source: ['/'],
+                target: '/gov/shanghai/wgj/:page',
```
Comment (TonyRL) [inline lib/v2/gov/radar.js:956]:
```suggestion
                target: '/gov/shanghai/wgj',
```

File: lib/v2/gov/shanghai/wgj/wgj.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    const params = {
+        'pageDoc.pageNo': page,
+    };
+    const response = await got({
+        method: 'post',
+        url: currentUrl,
+        headers: {
+            'Content-Type': 'application/x-www-form-urlencoded',
+        },
+        searchParams: queryString.stringify(params),
```
Comment (TonyRL) [inline lib/v2/gov/shanghai/wgj/wgj.js:21]: You can pass [an object to searchParams](https://github.com/sindresorhus/got/tree/v11.8.6#searchparams) without the need of query-string
```suggestion
        searchParams: {
            flag: 1,
            'pageDoc.pageNo': page,
        },
```

File: docs/government.md [hunk 3]
```diff
@@ -846,6 +846,11 @@ pageClass: routes
 
 </Route>
 
+### 上海市文旅局审批公告
+
+<Route author="gideonsenku" example="/gov/shanghai/wgj" path="/gov/shanghai/wgj/:page?" :paramsDesc="['页数，默认当前页']" radar="1">
+</Route>
```
Comment (TonyRL) [inline docs/government.md:852]: https://github.com/DIYgod/RSSHub/blob/eb523b33fb74c8a93dcf8dfd82bdfefeb2275de4/lib/v2/gov/shanghai/wgj/wgj.js#L11
```suggestion
<Route author="gideonsenku" example="/gov/shanghai/wgj" path="/gov/shanghai/wgj/:page?" :paramsDesc="['页数，默认第 1 页']" radar="1" />
```

File: lib/v2/gov/shanghai/wgj/wgj.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    const currentUrl = `${baseUrl}/shwgj_ywtb/core/web/welcome/index!toResultNotice.action`;
+    const page = ctx.params.page ?? 1;
+    const params = {
+        flag: 1,
+        'pageDoc.pageNo': page,
+    };
+    const response = await got({
+        method: 'post',
+        url: currentUrl,
+        searchParams: queryString.stringify(params),
```
Comment (TonyRL) [inline lib/v2/gov/shanghai/wgj/wgj.js:19]: https://github.com/DIYgod/RSSHub/pull/12862#discussion_r1271436769

**Comment by GideonSenku** [comment]
@TonyRL  我发现我的文档路由没有生效，`https://docs.rsshub.app/government.html#shang-hai-shi-ren-min-zheng-fu-shang-hai-shi-wen-lu-ju-shen-pi-gong-gao`，这个是生成的路由，真正的拼音应该是我写在文档中的：`https://docs.rsshub.app/government.html#shang-hai-shi-ren-min-zheng-fu-shang-hai-shi-wen-lv-ju-shen-pi-gong-gao`

是框架不认识 `lv`？把这个当作`lu`

**Pull request merged**